recipes/chimes-calculator/build.sh

```diff
@@ -0,0 +1,28 @@
+#!/usr/bin/env bash
+set -ex
+
+cmake --version
+
+cmake_options=(
+   "-DCMAKE_INSTALL_PREFIX=${PREFIX}"
+   "-DCMAKE_INSTALL_LIBDIR=lib"
```

Comment:
Those should be in the `CMAKE_ARGS` + some extras
```suggestion
   ${CMAKE_ARGS}
```

Reply:
@awvwgk Oh cool, thx, I was not aware of this conda feature (now I've found it...)